numpy/random/tests/test_extending.py

```diff
@@ -24,28 +26,28 @@
     import cython
 except ImportError:
     cython = None
+else:
+    cython_ver = cython.__version__.split('.')
+    if len(cython_ver) < 3 or cython_ver < ['0', '29', '14']:
```

Comment:
Can you add a "keep in sync with <that file that checks the cython version>" comment here?

Reply:
Tempting to use `LooseVersion` from distutils here too.

Reply:
Adding a comment.

Reply:
This was the file is was thinking of:

https://github.com/numpy/numpy/blob/8bc83b5fdcbd17ed2bc6de6e0903c19ca94f8797/tools/cythonize.py#L71-L77

I see you lifted the comment from there. I'd still be tempted to use `LooseVersion`, for both consistency, and making `0.29.14` greppable.

Reply:
ok